script.js

```diff
@@ -222,5 +222,23 @@ form.addEventListener('submit', (e) => {
     formErrorMsg.innerHTML = 'error';
   } else {
     formErrorMsg.innerHTML = 'success';
+
+    // preserve data to the local storage
+
+    const fullName = document.getElementById('full-name').value;
+    const message = document.getElementById('message').value;
+    const dataStorage = {
+      fullName,
+      email,
+      message,
+    };
+    localStorage.setItem('data', JSON.stringify(dataStorage));
```

Comment:
- Good job done so far 👍 👍. According to the project requirements, `When the user changes the content of any input field in the form the data object is updated and it is saved to the local storage`. The expected behavior should be that the local storage gets update as I change any of the input fields and not when I submit the form. Please fix this 😄